apis/anchore/swagger.yaml

```diff
@@ -1096,7 +1096,7 @@ paths:
         200:
           description: Registry configuration
           schema:
-            $ref: "#/definitions/RegistryConfiguration"
+            $ref: "#/definitions/RegistryConfigurationList"
```

Comment:
**Note: this is an API breaking change (single object versus array in JSON), so make sure it is properly communicated towards the users on release.**

Reply:
This is a fix for the swagger file of the anchore-engine. The anchore-engine response is a list. In the case of integrated services, we use "#/components/schemas/IntegratedServiceSpec" which is an object. So, our API descriptor isn't changed. (Nonetheless, we have to communicate it.)